### PR TITLE
Currency: Created build script and build_info.json for Black

### PR DESCRIPTION
**Pull request by aastha-sharma2**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: b/black/black_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    echo "Rust is already installed."
+fi
+
+#install necessary Python dependencies
+pip install -r test_requirements.txt
+pip install .
+pip install build
+
+#install
+if ! (pyproject-build) ; then
```
Comment (shubham-dayma-ibm) [inline b/black/black_ubi_9.3.sh:50]: Why are we building package?

File: b/black/black_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    echo "Rust not found. Installing Rust..."
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source "$HOME/.cargo/env"
+else
+    echo "Rust is already installed."
+fi
+
+#install necessary Python dependencies
+pip install -r test_requirements.txt
+pip install .
```
Comment (shubham-dayma-ibm) [inline b/black/black_ubi_9.3.sh:46]: I did see a case if package installation fails.

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]

File: b/black/black_ubi_9.3.sh (hunk 2 above)
Comment (aastha-sharma2) [reply b/black/black_ubi_9.3.sh]: > I did see a case if package installation fails.

Did changes as requested for the package if installation fails.

**Review by shubham-dayma-ibm — APPROVED** [review]
LGTM

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**